nomad/acl_endpoint.go

```diff
@@ -192,6 +192,62 @@ func (a *ACL) GetPolicies(args *structs.ACLPolicySetRequest, reply *structs.ACLP
 	return a.srv.blockingRPC(&opts)
 }
 
+// Bootstrap is used to bootstrap the initial token
+func (a *ACL) Bootstrap(args *structs.ACLTokenBootstrapRequest, reply *structs.ACLTokenUpsertResponse) error {
+	if done, err := a.srv.forward("ACL.Bootstrap", args, args, reply); done {
```

Comment:
Nothings ensuring it gets forwarded to authoritative region? If you aren't going to forward should at least verify that you are the authority?